src/modules/palnagotchi/pwngrid.h

```diff
@@ -0,0 +1,33 @@
+#ifdef CARDPUTER
+#include "ArduinoJson.h"
+#include "EEPROM.h"
+#include "M5Unified.h"
```

Comment:
Is it really needed?

Reply:
ArduinoJson is needed to load the payload, sent to pwngrid API.
Other libs are removed.